Assets/Scripts/Enemy/Boss/BossController.cs

```diff
@@ -28,6 +34,8 @@ public class BossController : MonoBehaviour
     public AudioClip damageTakenSFX;
     public float damageTakenSFXCooldown = 0.2f;
     private float lastDamageTakenSFXPlayTime = -Mathf.Infinity;
+    public float stopOverflowDamageNumbers = 2f;
+    private float ph;
```

Comment:
What does `ph` stand for? Please use a more descriptive name.

Reply:
it was a literal Place Holder name that I didnt change